scripts/readme-gen/templates/install_portaudio.tmpl.rst

```diff
@@ -0,0 +1,35 @@
+Install PortAudio
++++++++++++++++++
```

Comment:
Is this template needed in this repo?

I assume the template just sits there and does nothing, is committing such templates a common workflow?